fenix/app/src/main/java/org/mozilla/fenix/debugsettings/navigation/DebugDrawerRoute.kt

```diff
@@ -5,48 +5,63 @@
 package org.mozilla.fenix.debugsettings.navigation
 
 import androidx.annotation.StringRes
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
 import org.mozilla.fenix.R
-import org.mozilla.fenix.debugsettings.ui.DebugDrawerHome
+import org.mozilla.fenix.debugsettings.store.DebugDrawerAction
+import org.mozilla.fenix.debugsettings.store.DebugDrawerStore
+import org.mozilla.fenix.theme.FirefoxTheme
 import org.mozilla.fenix.debugsettings.tabs.TabTools as TabToolsScreen
 
 /**
  * The navigation routes for screens within the Debug Drawer.
  *
- * @property route The navigation route of the destination.
+ * @property route The unique route used to navigate to the destination.
```

Comment:
nit: could this be more explicit? I think we talked about potentially including the fact this could be a URI too?

Reply:
So I was wrong, it can't be a URI, but it can have data parameters. The API kdocs don't have anything meaningful to contribute, but I'll find something more illustrative to put. As it stands though, we'd probably need a future refactor to even support the data parameters.

I'll also update the `route` kdoc for the destination data class.